src/Components/TopBar/_styles.js

```diff
@@ -18,6 +18,15 @@ const styles = (theme) => ({
     menuButton: {
         marginRight: theme.spacing(2),
     },
+    localeButton: {
+        paddingTop: theme.spacing(1),
+        paddingBottom: theme.spacing(1),
+        paddingLeft: theme.spacing(2),
+        paddingRight: theme.spacing(2),
+        height: 'unset',
+        background: 'unset',
+        boxShadow: 'unset',
```

Comment:
There are a set of custom styles that are applied on the Material UI Theme:
[/src/theme.js](https://github.com/cfrancisco/gui-builder/blob/dev/src/theme.js#L29)
In the specific case of the buttons, there is the following overrides:
`
MuiButton: {
    root: {
        background: 'linear-gradient(45deg, #6b91fe 30%, #53c7ff 90%)',
        height: 48,
        boxShadow: '0 3px 5px 2px rgba(105, 137, 255, 0.3)',
    },
},
`
It might be interesting to review these style overrides on 'MuiButton.root'.